server/routes/auth_routes.py

```
@@ -29,6 +34,43 @@ def _name_to_slug(name: str) -> str:
+    try:
+        email_svc = get_email_service()
+    except ValueError:
+        with db_pool.get_admin_connection() as c:
+            with c.cursor() as cur:
+                cur.execute("UPDATE users SET email_verified = TRUE WHERE id = %s", (user_id,))
+                c.commit()
+        return True
+
+    code = f"{secrets.randbelow(1000000):06d}"
+    code_hash = hashlib.sha256(code.encode()).hexdigest()
+    expires = datetime.now() + timedelta(minutes=VERIFICATION_CODE_EXPIRY_MINUTES)
```

Ruff (0.15.20)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)